magicctapipe/scripts/lst1_magic/semi_automatic_scripts/clusters.py

```diff
@@ -0,0 +1,41 @@
+"""
+Module for generating bash script lines for running analysis in different clusters
+"""
+
+
+def slurm_lines(p, J, array=None, mem=None, out_err=None):
+    """
+    Function for creating the general lines that slurm scripts are starting with.
+
+    Parameters
+    ----------
+    p : str
+        Name of the queue
+    J : str
+        Job name
+    array : None or int
+        If not none array of jobs from 0 to array will be made
+    mem : None or str
+        Requested memory
```

Comment:
I would add `If None, no specific memory will be requested`. Also, may an example of what can be specified e.g. "16g".

Reply:
this is actually no true, if you do not specify the this parameter the default of the cluster setting is used, which I think is 5 GB